backend/substrapp/compute_tasks/errors.py

```diff
@@ -61,13 +62,21 @@ class _ComputeTaskError(RuntimeError):
 
 
 class BuildError(_ComputeTaskError, CeleryRetryError):
```

Comment:
The orchestrator [will fail the task on report registration](https://github.com/Substra/orchestrator/blob/2a6de8e71e287fcbe8e21e35b125e4819d15c0ff/lib/service/failure_report.go#L58), yet this is a `CeleryRetryError` subclass. We should only send the report on the last unsuccessful attempt.

Reply:
It is sent only on the last unsuccessful attempt because `_store_failure()` is called only in the `ComputeTask.on_failure()` method.